docs/src/content/en/docs/frameworks/agentic-uis/copilotkit.mdx

```diff
@@ -11,139 +11,139 @@ import Image from "next/image";
-This installs the core CopilotKit runtime needed for backend integration with Mastra.
-All commands install the same @copilotkit/runtime package but use different package manager syntax.
-*/}
+Create Mastra project.
 
-<Tabs items={["npm", "yarn", "pnpm"]}>
+<Tabs items={["npx", "npm", "yarn", "pnpm"]}>
+  <Tabs.Tab>
+    ```bash copy filename="terminal"
+    npx create-mastra@latest
+    ```
+  </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    npm install @copilotkit/runtime
+    ```bash copy filename="terminal"
+    npm create mastra mastra-server
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    yarn add @copilotkit/runtime
+    ```bash copy filename="terminal"
+    yarn create mastra mastra-server
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    pnpm add @copilotkit/runtime
+    ```bash copy filename="terminal"
+    pnpm create mastra mastra-server
     ```
   </Tabs.Tab>
 </Tabs>
 
-## Set up the runtime
-
-You can leverage Mastra's custom API routes to add CopilotKit's runtime to your Mastra server.
-
-The current version of the integration leverages `MastraClient` to format Mastra agents into the AGUI format of CopilotKit.
-
-{/*
-LLM CONTEXT: This Tabs component shows installation commands for the Mastra AGUI package.
-Each tab displays the installation command for that specific package manager.
-This installs the alpha version of @ag-ui/mastra which provides CopilotKit integration capabilities.
-All commands install the same @ag-ui/mastra package but use different package manager syntax.
-*/}
+Install additional packages.
 
 <Tabs items={["npm", "yarn", "pnpm"]}>
   <Tabs.Tab>
-    ```bash copy
-    npm install @ag-ui/mastra
+    ```bash copy filename="terminal"
+    npm install @copilotkit/runtime @ag-ui/mastra
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    yarn add @ag-ui/mastra
+    ```bash copy filename="terminal"
+    yarn add @copilotkit/runtime @ag-ui/mastra
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    pnpm add @ag-ui/mastra
+    ```bash copy filename="terminal"
+    pnpm add @copilotkit/runtime @ag-ui/mastra
     ```
   </Tabs.Tab>
 </Tabs>
 
-Next, let's update the Mastra instance with a custom API route for CopilotKit.
+Configure your Mastra instance to include CopilotKit's runtime endpoint with an optional typed runtime context:
 
-```typescript filename="src/mastra/index.ts" showLineNumbers copy
+```typescript filename="src/mastra/index.ts" showLineNumbers copy {7-12,23-41}
 import { Mastra } from "@mastra/core/mastra";
 import { PinoLogger } from "@mastra/loggers";
 import { LibSQLStore } from "@mastra/libsql";
-import { CopilotRuntime, copilotRuntimeNodeHttpEndpoint, ExperimentalEmptyAdapter } from "@copilotkit/runtime";
 import { registerCopilotKit } from "@ag-ui/mastra";
 import { weatherAgent } from "./agents/weather-agent";
 
-const serviceAdapter = new ExperimentalEmptyAdapter();
+// Define your runtime context type
```

Comment:
General comment. Remove comments from the code examples. If there is something needs to be explained, consider adding a paragraph that explains what the code block does. Or, perhaps consider adding bullet points below the code block with any required explanations.